tests/test_configure_feedstock.py

```diff
@@ -1941,6 +1942,89 @@ def test_get_used_key_values_by_input_order(
             squished_input_variants,
             squished_used_variants,
             all_used_vars,
-        )
+        )[0]
```

Comment:
I think it'd be cleaner to rewrite this not as comparing some unspecified 0th element, but be more explicit it à la:
```diff
@@ -1935,16 +1935,16 @@ def test_get_used_key_values_by_input_order(
     squished_input_variants,
     squished_used_variants,
     all_used_vars,
-    used_key_values,
+    expected_key_values,
 ):
-    assert (
+    used_key_values, _ = (
         configure_feedstock._get_used_key_values_by_input_order(
             squished_input_variants,
             squished_used_variants,
             all_used_vars,
-        )[0]
-        == used_key_values
+        )
     )
+    assert used_key_values == expected_key_values
```

Reply:
Thanks for the comment! Done!